### PR TITLE
MAT-8163 Remove Feature Flags

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-8163](https://jira.cms.gov/browse/MAT-8163)
(Optional) Related Tickets:
https://github.com/MeasureAuthoringTool/madie-util/pull/119
https://github.com/MeasureAuthoringTool/madie-cql-library/pull/173
https://github.com/MeasureAuthoringTool/madie-layout/pull/389
https://github.com/MeasureAuthoringTool/madie-measure/pull/779
https://github.com/MeasureAuthoringTool/madie-editor/pull/454

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/CqlBuilderPanel/CqlBuilderSectionPanelNavTabs.tsx [hunk 1]
```diff
@@ -3,23 +3,11 @@ import { Tabs, Tab } from "@madie/madie-design-system/dist/react";
...
-  const {
-    activeTab,
-    setActiveTab,
-    CQLBuilderParameters,
-    CQLBuilderFunctions,
-    QICoreCodeSearch,
-    QICoreValueSetSearch,
-    isQDM,
-  } = props;
+  const { activeTab, setActiveTab, isQDM } = props;
```
Comment (adongare) [inline src/CqlBuilderPanel/CqlBuilderSectionPanelNavTabs.tsx]: isQDM can be removed as well since this is not being used. and from line 6.

File: src/CqlBuilderPanel/CqlBuilderPanel.tsx [hunk 2]
```diff
@@ -44,12 +44,7 @@ export default function CqlBuilderPanel({
   makeExpanded,
 }) {
   const featureFlags = useFeatureFlags();
-  const {
-    CQLBuilderParameters,
-    CQLBuilderFunctions,
-    QICoreCodeSearch,
-    QICoreValueSetSearch,
-  } = featureFlags;
+  // const {} = featureFlags;
```
Comment (adongare) [inline src/CqlBuilderPanel/CqlBuilderPanel.tsx]: line 46 & 47 along with its import not required anymore

File: src/CqlBuilderPanel/CqlBuilderPanel.test.tsx
Comment (adongare) [inline src/CqlBuilderPanel/CqlBuilderPanel.test.tsx:1]: I think there are more test cases that use these flags need to be updated

File: src/madie-madie-util.d.ts [hunk 3]
```diff
@@ -1,12 +1,7 @@
 declare module "@madie/madie-util" {
   import { LifeCycleFn } from "single-spa";
 
-  interface FeatureFlags {
-    CQLBuilderParameters: boolean;
-    CQLBuilderFunctions: boolean;
-    QICoreCodeSearch: boolean;
-    QICoreValueSetSearch: boolean;
-  }
+  interface FeatureFlags {}
```
Comment (adongare) [inline src/madie-madie-util.d.ts]: do we need this empty interface?

**Review by adongare — APPROVED** [review]
